README.md

```diff
@@ -0,0 +1,54 @@
+<p align="center">
+    <a href="https://cs.symfony.com">
+        <img src="./logo.png" title="PHP CS Fixer" alt="PHP CS Fixer logo">
+    </a>
+</p>
+
+PHP Coding Standards Fixer
+==========================
+
+The PHP Coding Standards Fixer (PHP CS Fixer) tool fixes your code to follow standards;
+whether you want to follow PHP coding standards as defined in the PSR-1, PSR-2, etc.,
+or other community driven ones like the Symfony one.
+You can **also** define your (team's) style through configuration.
+
+It can modernize your code (like converting the ``pow`` function to the ``**`` operator on PHP 5.6)
+and (micro) optimize it.
+
+If you are already using a linter to identify coding standards problems in your
+code, you know that fixing them by hand is tedious, especially on large
+projects. This tool does not only detect them, but also fixes them for you.
+
+## Installation
+
+See [installation instructions](./doc/installation.rst).
+
+## Documentation
+
+* [Usage](./doc/usage.rst)
```

Comment:
I think it would be good to have a very simpel command example, like `php-cs-fixer fix ./src` or something and than link to the doc file.

Reply:
[I reworked the file a bit](https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/9e95ec0524429d1ef363db2e5c5da8f696bffe13/README.md), please let me know what you think.

Reply:
looks very good  :)